SearchEngines.yml

```diff
@@ -74,6 +82,13 @@ Aguea:
     params:
       - q
     backlink: 's.py?q={k}'
+Allaverksamheter:
+  - 
+    urls:
+      - www.allaverksamheter.se
```

Comment:
Yes it's a business register similar to yellow pages,
companies do have websites listed so it will link from the search result as such.